src/foam/layout/SectionAxiom.js

```diff
@@ -23,6 +23,9 @@ foam.CLASS({
     {
       name: 'subTitle'
     },
+    {
+      name: 'wizardTitle'
```

Comment:
@kgrgreer is this better as `wizardTitle`, `shortTitle`, or `navTitle`?

Reply:
personally, i prefer navTitle if it is being used on the navigation portion.

Reply:
Sections are more generic than wizards, so I agree with Kenny, something like navTitle or even just 'label' would be better.